src/test/java/org/fundacionjala/sfdc/stepdefinition/ContactStepsCheck.java

```diff
@@ -1,103 +1,33 @@
 package org.fundacionjala.sfdc.stepdefinition;
 
 import cucumber.api.java.en.And;
-import cucumber.api.java.en.Given;
-import cucumber.api.java.en.Then;
-import cucumber.api.java.en.When;
-import org.fundacionjala.sfdc.pageobjects.SalesForceTabClassic;
 import org.fundacionjala.sfdc.pageobjects.contacts.ContactInputs;
-import org.fundacionjala.sfdc.pageobjects.contacts.SFCDetailsPage;
-import org.fundacionjala.sfdc.pageobjects.contacts.SFCMainPage;
 import org.fundacionjala.sfdc.pageobjects.contacts.SFCNewModifyPage;
-import org.testng.Assert;
+import org.fundacionjala.sfdc.pageobjects.opportunities.SFONewModifyPage;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.fundacionjala.sfdc.pageobjects.opportunities.SFONewModifyPage'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)